.ci/pytorch/windows/arm64/bootstrap_buildtools.bat

```diff
@@ -18,31 +18,49 @@ curl -L -o "%INSTALLER_FILE%" %DOWNLOAD_URL%
 echo Installing Visual Studio Build Tools with C++ components...
 echo Installing MSVC %MSVC_VERSION%
 if "%MSVC_VERSION%" == "latest" (
-    "%INSTALLER_FILE%" --norestart --nocache --quiet --wait --installPath "%DEPENDENCIES_DIR%\VSBuildTools" ^
+    "%INSTALLER_FILE%" --norestart --quiet --wait --installPath "%DEPENDENCIES_DIR%\VSBuildTools" ^
+        --add Microsoft.VisualStudio.Workload.VCTools ^
+        --add Microsoft.VisualStudio.Component.Windows10SDK ^
         --add Microsoft.VisualStudio.Component.Windows11SDK.22621 ^
         --add Microsoft.VisualStudio.Component.VC.ASAN ^
         --add Microsoft.VisualStudio.Component.VC.CMake.Project ^
+        --add Microsoft.VisualStudio.Component.VC.CoreBuildTools ^
+        --add Microsoft.VisualStudio.Component.VC.CoreIde ^
+        --add Microsoft.VisualStudio.Component.VC.Redist.14.Latest ^
+        --add Microsoft.VisualStudio.Component.VC.Tools.ARM64EC ^
         --add Microsoft.VisualStudio.Component.VC.Tools.ARM64 ^
         --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64
 ) else if "%MSVC_VERSION%" == "14.40" (
```

Comment:
Why do you need to support so many different versions of the toolchain right away? For CI you can always pick one, can't you?

Reply:
Since we are not testing those, do you mind deleting dead code from the script?
Also, I would advice against latest, as this would results in CD builds starting to fail on one day and it makes it really hard to figure out that it was due to the version change